doc-surrealdb_versioned_docs/version-2.x/surrealql/statements/create.mdx

```diff
@@ -136,6 +136,32 @@ CREATE townsperson, cat, dog SET
 ]
 ```
 
+The `||` syntax can be used to create a specified number of records. To use this syntax, add a colon and the number of records to create after the table name.
```

Comment:
```suggestion
The `| |` syntax can be used to create multiple records in a single execution. The number that follows the table name specifies the number of records created. 

To use this syntax, add a colon and the number of records to create after the table name.
```

Reply:
@Dhghomon what do you think? RE: Suggestion

Reply:
Looks good!